### DISCUSSION TITLE
Unranking mod

### DISCUSSION THREAD

**Discussion by MineTurtlee** [ppy/osu#24149] [Ideas]
It would be like ScoreV2 in osu!stable but there is no ScoreV2. 
Aiming: For **PRACTICE PURPOSES**

and please stop thumb down pls

**Comment by MishterKirby** [comment]
What use would this even serve?

**Comment by SherlockWh0** [comment]
just log out lol

**Reply by MineTurtlee** [reply]
As **logging out** also unrank, but it also takes time to **log in**, if you want you can just without logging out so there's the mod.

> And if we login after playing, that would take a sum of 10 seconds.

**Comment by cihe13375** [comment]
An alternative direction (which I believe makes more sense and has more benefits) is to make the "appear offline" mode read-only, i.e. one can read messages, download beatmaps, etc. in this mode, but cannot send messages to public channels, submit scores etc. After all, it doesn't make sense for an "offline" user to submit scores or send public messages, and doing so is effectively leaking their status.

**Reply by MineTurtlee** [reply]
Hmmm maybe but umm, peppy or the devs didn't think about it

**Comment by Rekunan** [comment]
This follows same direction of #17121 so I'd consider it a duplicate.
